docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,59 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement to have a reliable service that would
+provide a reference text for individual license identifiers, exception identifiers or license expression emerged.
+The one with a stable URL schema, that is not susceptible to the link rot, so that we can safely use it to provide
+backwards compatibility for older clients.
+
+[`licenses.nuget.org`](https://licenses.nuget.org) fulfills that role. Gallery uses it to provide the license
+text reference for the pacakages that specify their license using license expression and `nuget.exe pack` operation
+generates links to `licenses.nuget.org` for the [`licenseUrl`](nuspec.md#licenseurl) element to provide backwards
+compatibility with older clients that don't support the `license` element.
+
+## Protocol
+
+`licenses.nuget.org` is intended to be viewed by people in their browsers, no machine-readable responses are provided.
```

Comment:
I don't think we should put backticks around licenses.nuget.org. We don't do this for nuget.org so why is this different?